offload/test/mapping/data_member_ref.cpp

```diff
@@ -60,7 +60,8 @@ int main() {
   printf("Host %d %d.\n", Bar.VRef.Data, V.Data);
   // CHECK: Host 123456.
   printf("Host %d.\n", *Baz.VRef.Data);
-#pragma omp target map(*Baz.VRef.Data) map(from : D1, D2)
+#pragma omp target map(Baz.VRef.Data) map(*Baz.VRef.Data) map(V1.Data[0 : 0])  \
```

Comment:
For pointer-attachment on `V1.Data` to trigger, we need to `map(V1.Data[0:0])`. Otherwise, the implicit `map(V1)` will cause `V1.Data` to retain the host pointer in the target region.